templates/CRM/common/CMSPrint.tpl

```diff
@@ -35,14 +35,14 @@
 {if isset($browserPrint) and $browserPrint}
 {* Javascript window.print link. Used for public pages where we can't do printer-friendly view. *}
 <div id="printer-friendly">
-<a href="#" onclick="window.print(); return false;" title="{ts}Print this page.{/ts}">
+<a href="#" onclick="window.print(); return false;" title="{ts}Print this page.{/ts}" aria-label="{ts}Print this page.{/ts}">
```

Comment:
@monishdeb this has other changes in it as well though? I was hoping just to merge the file duplication clean up since I haven't formed an opinion on the other changes as yet